lpcompiler.pas

```diff
@@ -1653,13 +1662,17 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
               addMethod(OldDeclaration as TLapeGlobalVar);
             end;
 
+            MethodDef := FuncHeader.MethodDef;
```

Comment:
Ah, I didn't realize this was part of a with-statement! I thought this referred to the variable at line 1300.